### PR TITLE
Add arm32 test

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

File: .jenkins/JenkinsfileArm [hunk 1]
```diff
@@ -39,15 +39,45 @@ node {
...
             sh 'cmake -H./ -Bout -DESCARGOT_MODE=release -DESCARGOT_THREADING=ON -DESCARGOT_TEMPORAL=ON -DESCARGOT_TCO=ON -DESCARGOT_TEST=ON -DESCARGOT_OUTPUT=shell -GNinja'
         }
 
-        stage('Build') {
+        stage('Build on aarch64') {
             sh 'cmake --build ./out/'
         }
 
-        stage('Running test') {
+        stage('Running test on aarch64') {
```
Comment (clover2123) [inline .jenkins/JenkinsfileArm:80]: It seems like that tests on `arm` and `aarch64` are done in sequential.
But, is it possible to run the both test in parallel?

Comment (ksh8281) [reply .jenkins/JenkinsfileArm:80]: Good point. I can fix it

File: .jenkins/JenkinsfileArm [hunk 2]
```diff
@@ -39,17 +39,48 @@ node {
...
-                sh '#!/bin/bash\nGC_FREE_SPACE_DIVISOR=1 tools/run-tests.py --engine="${WORKSPACE}/out/escargot" --extra-arg="--skip intl402" new-es v8 spidermonkey chakracore test262'
+            timeout(60) {
+                parallel (
+                    'arm32' : {
+                        sshCommand remote: arm32RemoteInfo, command: "\
+                        cd escargot;\
+                        GC_FREE_SPACE_DIVISOR=1 tools/run-tests.py --engine='/root/escargot/out/escargot' --extra-arg='--skip intl402' new-es v8 spidermonkey chakracore test262"
+                    },
+                    'aarch64' : {
+                        sh '#!/bin/bash\nGC_FREE_SPACE_DIVISOR=1 tools/run-tests.py --engine="${WORKSPACE}/out/escargot" --extra-arg="--skip intl402" new-es v8 spidermonkey chakracore test262'
```
Comment (clover2123) [inline .jenkins/JenkinsfileArm:81]: Question) Why do you add shebang(`#!`) along with a notation of bash here?
Is this really necessary?

Comment (ksh8281) [reply .jenkins/JenkinsfileArm:81]: We need to use bash for running chakaracore test.

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**